index.php

```diff
@@ -1,4 +1,10 @@
 <?php
 
+ini_set('display errors', 1);
+error_reporting(E_ALL);
+
+define('ROOT', dirname(__FILE__));
```

Comment:
Create App singleton and create static getRootPath method, which will return dirname(__FILE__).
And instead of ROOT const use App::getRootPath()